### ISSUE TITLE
SDL throws on waking up from hibernation

### ISSUE THREAD

**Issue by gleblebedev** [rokups/rbfx#277]
MediaDevice::GetDefaultAudioRenderId(AudioDeviceRole::Default); fails

Exception thrown at 0x00007FFE35723B29 (KernelBase.dll) in PsyRadioPlayer.UWP.exe: WinRT originate error - 0x80070490 : 'Element not found.'.
Exception thrown at 0x00007FFE35723B29 in PsyRadioPlayer.UWP.exe: Microsoft C++ exception: Platform::COMException ^ at memory location 0x0000005D0C2FF958. HRESULT:0x80070490 Element not found.
WinRT information: Element not found.

 	KernelBase.dll!00007ffe35723b29()	Unknown
 	vcruntime140_app.dll!00007ffe0b856220()	Unknown
 	vccorlib140_app.dll!00007ffe0005a58c()	Unknown
 	Urho3D.dll!__abi_WinRTraiseException(long __hrArg) Line 1132	C++
 	[Inline Frame] Urho3D.dll!__abi_ThrowIfFailed(long) Line 100	C++
>	Urho3D.dll!WASAPI_ActivateDevice(SDL_AudioDevice * _this, const SDL_bool isrecovery) Line 207	C++
 	Urho3D.dll!RecoverWasapiDevice(SDL_AudioDevice * this) Line 252	C
 	Urho3D.dll!WASAPI_WaitDevice(SDL_AudioDevice * this) Line 314	C
 	Urho3D.dll!SDL_RunAudio(void * devicep) Line 758	C
 	Urho3D.dll!SDL_RunThread(void * data) Line 283	C
 	Urho3D.dll!RunThread(void * data) Line 91	C

**Comment by gleblebedev** [comment]
This is a low-priority bug and probably should be open on SDL, not rbfx

**Comment by rokups** [comment]
3bbeaaeef253d00ea02edda2b0d3719b0d09bf0e solves this?

**Comment by gleblebedev** [comment]
> [3bbeaae](https://github.com/rbfx/rbfx/commit/3bbeaaeef253d00ea02edda2b0d3719b0d09bf0e) solves this?

No, it's a different one. Actually I have to test it again since audio code changed. Maybe it isn't an issue anymore.
